.circleci/config.yml

```diff
@@ -38,7 +38,8 @@ jobs:
       - run:
           name: Update staging branch
           command:
-            git push git@github.com:artsy/kaws.git $CIRCLE_SHA1:staging --force
+            git push git@github.com:artsy/kaws.git
+            $CIRCLE_SHA1:refs/heads/staging --force
```

Comment:
Is the linebreak intentional? Looks broken to me.

Reply:
Not intentional--prettier maybe? I'm also seeing the same on line [32 & 33](https://github.com/artsy/kaws/blob/2076f9916b562d1bb127b3a4360b1ec436d4c96f/.circleci/config.yml#L32-L33)